script/php/dataScrape.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+/**
+ * Creates the connection from the database `pokedex`.
+ *
+ * @return PDO $db is the database.
+ */
+function dbConn() : PDO {
+    return new PDO('mysql:host=127.0.0.1; dbname=pokedex', 'root');
+}
+
+/**
+ * This is an API call to the pokeAPI and pulls all of the data from 151 pokemon and puts them in an array. The sleep stops the api call half way through for 60 seconds
+ * $temp stores each Pokemon data for a limited time, allowing for the needed data to be removed and stored in pokemon before being discarded on the next loop.
+ *
+ * @return array $pokemon - holds all of our pokemon, with the only data we need from each, name, type 1 and type 2.
+ */
+function grabApi() : array {
+    $pokemon = [];
+    $curl = curl_init();
+    curl_setopt($curl, CURLOPT_RETURNTRANSFER, 1);
+    $i = 0;
+    while ($i < 151) {
+        if ($i === 75){
+            sleep(60);
+        }
+        $i++;
+        curl_setopt($curl, CURLOPT_URL, 'https://pokeapi.co/api/v2/pokemon/' . $i . '/');
+        $resp = curl_exec($curl);
+        $temp = json_decode($resp, TRUE);
+        $name = $temp['forms'][0]['name'];
+        if (isset($temp['types'][1]['type']['name'])) {
+            $type1 = $temp['types'][0]['type']['name'];
+            $type2 = $temp['types'][1]['type']['name'];
+            $pokemon[] = [$name, $type1, $type2];
+        } else {
+            $type1 = $temp['types'][0]['type']['name'];
```

Comment:
this is a repeat of line 33, if this is happening both times then pull this out the if statement and only do the assignment of type2 and the pokemon array in the if/else statement.